students/Shweta/Lesson03/list_lab.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+#Series 1
+#Create and do modifications on list
+#including pop, append, +,insert
+#
+
+print("--------Series 1 started--------")
+fruit_list=["Apples","Pears","Oranges","Peaches"]
```

Comment:
Missing whitespace around assignment operator - here as well as everywhere in file